src/debugBridge.ts

```diff
@@ -64,7 +65,7 @@ brokerSocket.on('message', (data: Buffer) => {
             log('debuggerSocket: connection established with main lambda process');
         });
     }
-
+    log('from broker: ', data.toString('utf8').substring(0, 50));
```

Comment:
it might be better to put these log statements into if block:
```js
if (LOGS_ENABLED === 'true') {
     log('from broker: ', data.toString('utf8').substring(0, 50));
}
```
So there won't be extra overhead for `data.toString('utf8').substring(0, 50)` when logging is disabled